docs/docs/azure.md

```diff
@@ -41,28 +38,23 @@ question mark is the size. Other distributions should work as long as they are o
     * Pick a size that's got plenty of RAM. For example, you might want to click "Change size" to find `DC4s_v3` type
     * Ensure that the public inbound ports are open for SSH access
 
-    ![](images/create-d4sv3-vm.png)
+    ![](images/DC4S_V3 Screenshot.png)
     ![](images/create-d4sv3-vm-publicinbound.png)
 
 After you have logged on to the VM:
 
-1. Check that the `enclave` device is present in the `/dev/sgx/` or `/dev/sgx_*` directory
-2. Check driver version `dmesg | grep sgx`. Conclave requires driver version 1.33+
-3. If either check fails:
-    * Download the [driver](https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntu20.04-server/)
-    * Follow the [install instructions](https://download.01.org/intel-sgx/sgx-dcap/1.8/linux/docs/Intel_SGX_DCAP_Linux_SW_Installation_Guide.pdf)
-4. Add your user into `sgx_prv` group to give it access to SGX.
+1. Check that the `enclave` device is present in the `/dev/sgx/` directory.
+2. Add your user into `sgx_prv` group to give it access to SGX.
 
 ```sh
 sudo usermod -aG sgx_prv $USER
 ```
-5. Log out from the VM and log in again.
+3. Log out from the VM and log in again.
 
 You have set up an Azure VM with the latest attestation protocols.
```

Comment:
@shamsasari @anien  While reading this documents, I did not understand why we mention attestation in the first part but then we do not actually install it and we tell the user to "skip ... if you are only looking for a quick setup"

I do not have the full context of this page, but at this specific line of these instructions we do not have attestation available, wouldn't be better to say something like
```
You have set up an Azure VM with SGX.
If you also want to setup attestation, please follow the instructions below.

Reply:
Agree. When we have a new user coming to quickly setup Azure VM and run for example hello world sample, we should keep this short. Especially as we advice to use DC4s_v3 images that are ready to go.

Reply:
Moved the entire DCAP section to another page.